write/command.go

```diff
@@ -18,7 +19,7 @@ import (
 func (o Options) Run() error {
 	in, _ := stdin.Read()
 	if in != "" && o.Value == "" {
-		o.Value = in
+		o.Value = strings.Replace(in, "\r", "", -1)
```

Comment:
Just a minor nitpick comment, otherwise looks great!

```suggestion
		o.Value = strings.ReplaceAll(in, "\r", "")
```

Reply:
There wasn't a specific reason. I'm just quite new to go (:

Thanks for pointing it out.

Reply:
Awesome! Welcome to Go, and thank you for all the contributions ❤️